api-specificatie/openapi.yaml

```diff
@@ -769,6 +702,19 @@ components:
       schema:
         type: string
 
+    locatie-query:
+      description: Coordinaten van een locatie die als query-parmeter gebruikt wordt om een object te zoeken. Let op, explode is false dus het formaat is ?locatie=1.2345,5.4321
+      name: locatie
+      in: query
+      required: false
+      explode: false
+      schema:
+        type: array
+        minItems: 2
```

Comment:
@strijm is het ook mogelijk een punt op te geven met meer dan 2 coordinaten?

Reply:
Goede vraag. Ik ben zo dicht mogelijk bij de definitie van een point  gebleven conform opengis. 
http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml 

Neemt niet weg dat we dat aan kunnen scherpen indien mogelijk.

Reply:
Een punt kan bij BAG in theorie ook 3D zijn, dus ja.